homeassistant/components/hassio/services.yaml

```
@@ -116,6 +121,11 @@ backup_partial:
       example: "password"
       selector:
         text:
+    compressed:
+      name: Compressed
+      description: Optionally use compressed archives (default = true)
+      selector:
```

Sets the default, and removed `default` from the text (as it's already displayed it is optional).

```suggestion
      description: Use compressed archives
      default: true
      selector:
```